Dockerfile

```diff
@@ -20,27 +18,18 @@ RUN useradd -d /hubot -m -s /bin/bash -U hubot
 USER  hubot
 WORKDIR /hubot
 
-# Install hubot
+# Install Hubot
 RUN yo hubot --owner="MyDrive Systems <systems@mydrivesolutions.com>" --name="DriveBot" --description="Your plastic pal who's fun to be with" --defaults
 
 # Some adapters / scripts
-RUN npm install hubot-slack --save && npm install
-RUN npm install hubot-pugme --save && npm install
-RUN npm install hubot-standup-alarm --save && npm install
-RUN npm install hubot-google-images --save && npm install
-RUN npm install hubot-redis-brain --save && npm install
-RUN npm install signalfx --save && npm install
-# RUN npm install hubot-auth --save && npm install
-# RUN npm install hubot-google-translate --save && npm install
-# RUN npm install hubot-auth --save && npm install
-# RUN npm install hubot-github --save && npm install
-# RUN npm install hubot-alias --save && npm install
-# RUN npm install hubot-gocd --save && npm install
-# RUN npm install hubot-youtube --save && npm install
-# RUN npm install hubot-s3-brain --save && npm install
+RUN npm install hubot-slack --save
+RUN npm install hubot-pugme --save
+RUN npm install hubot-standup-alarm --save
+RUN npm install hubot-google-images --save
+RUN npm install hubot-redis-brain --save
+RUN npm install signalfx --save
```

Comment:
do we need these on different lines, we could probably just make this `npm install --save hubot-slack hubot-pugme hubot-standup-alarm ...`